artifactory/cli.go

```diff
@@ -1844,6 +1844,14 @@ func getReleaseBundleDistributeFlags() []cli.Flag {
 			Name:  "country-codes",
 			Usage: "[Default: '*'] Semicolon-separated list of wildcard filters for site country codes. ` `",
 		},
+		cli.BoolFlag{
+			Name:  "sync",
+			Usage: "[Defailt: false] Set to true to enable sync distribution. ` `",
```

Comment:
I suggeat adding the following to the end of the description:

(the command execution will end when the distribution process ends).